Documents/FunctionalSpecifications/functionalSpecifications.md

```diff
@@ -191,106 +193,137 @@ It uses the monospace sans-serif font in uppercase, the same font as the game Pa
 
 #### Timer
 
 A special mechanic to the game is the timer. Each time you try to get to the finish line, the player has a limited period of time. It is shown at the bottom of the screen, using a text to tell that it is the timer with the yellow font, and a green bar that decreases in length to tell how many time is left before loosing.
 
-<img src="Images/froggerTimer.png" width="80%">
+<img src="Images/froggerTimer.png" width="60%">
 
 The timer lasts for 30 seconds, translated into 60 "beats" or "ticks" (the term beats will be used for the rest of the document). Every time the player gets to the finish line before the end of the timer, a message in the middle of the screen tells how many beats was left, which then gives certain number of points (described in [Scoring](#scoring)).
 
-<i style="font-size: 10px">Example of the time left message, here the player got to the finish line 26 beats before the end of the timer</i><br>
-<img src="Images/froggerTimeLeft.png" width="40%">
+<i style="font-size: 12px">Example of the time left message, here the player got to the finish line 26 beats before the end of the timer</i><br>
+<img src="Images/froggerTimeLeft.png" width="20%">
 
 ### Environment
 ---
 
 The game is created in two times: first there is the background, then we add the different sprites that moves on the screen.
 
-<i style="font-size: 10px">First image is only the background, the second is with the moving sprites</i><br>
+<i style="font-size: 12px">First image is only the background, the second is with the moving sprites</i><br>
 <img src="Images/froggerBackground.png" width="35%">
 <img src="Images/froggerBackgroundAndSprites.png" width="35%">
 
 #### Finish line
 
 The finish line is described by the green area at the top:
 
-<img src="Images/froggerFinishLine.png" width="80%"><br>
+<img src="Images/froggerFinishLine.png" width="60%"><br>
 
 To win the level, the player has to go on each of the five water cells, and not on the grass.
 
-<i style="font-size: 10px">Every animation steps when stepping on a finishing cell</i><br>
-<img src="Images/froggerArriving.png" width="80%">
+<i style="font-size: 12px">Every animation steps when stepping on a finishing cell</i><br>
+<img src="Images/froggerArriving.png" width="60%">
 
 #### Insects
 
 Sometimes, an insect can spawn on one water cell of the finish line.
 
-<img src="Images/froggerInsect.png" width="35%"><br>
+<img src="Images/froggerInsect.png" width="20%"><br>
 
-When going on it, you gain an extra 200 points on the score.
+When going on it, the player gain extra points (see [points table](#scoring)).
 
 #### Lady frogs
 
-As well as the insect, a lady frog can spawn 
+As well as the insect, a lady frog can spawn randomly.
+
+<img src="Images/froggerLadyFrog.png" width="20%"><br>
+
+When the player steps on it, he grabs the lady frog, to escort it to the finish line. If the player successfully escorted the lady frog, the player gain extra points (see [points table](#scoring)).
```

Comment:
the player gain*s*

Reply:
fixed and i found the same mistake some lines before that i also fixed